portfolio/src/main/java/com/google/sps/objects/User.java

```diff
@@ -0,0 +1,42 @@
+package com.google.sps.objects;
+
+import java.util.Objects;
+
+public final class User {
```

Comment:
Javadoc is required for public classes. The methods here are self-explanatory, but the class itself should have a comment, and optionally the member variables.

https://google.github.io/styleguide/javaguide.html#s7.3-javadoc-where-required